x-pack/plugins/security_solution/public/common/components/event_details/get_alert_summary_rows.tsx

```diff
@@ -114,19 +221,24 @@ export const getSummaryRows = ({
   eventId: string;
   isDraggable?: boolean;
 }) => {
-  const eventCategoryField = find({ category: 'event', field: 'event.category' }, data);
-
-  const eventCategory = Array.isArray(eventCategoryField?.originalValue)
-    ? eventCategoryField?.originalValue[0]
-    : eventCategoryField?.originalValue;
+  const eventCategories = getEventCategoriesFromData(data);
 
   const eventCodeField = find({ category: 'event', field: 'event.code' }, data);
 
   const eventCode = Array.isArray(eventCodeField?.originalValue)
     ? eventCodeField?.originalValue?.[0]
     : eventCodeField?.originalValue;
 
-  const tableFields = getEventFieldsToDisplay({ eventCategory, eventCode });
+  const eventRuleTypeField = find({ category: 'kibana', field: 'kibana.rule.type' }, data);
+  const eventRuleType = Array.isArray(eventRuleTypeField?.originalValue)
+    ? eventRuleTypeField?.originalValue?.[0]
```

Comment:
should this be `.[0]` or just `[0]`?

Reply:
Ah yes, it did look odd to me as well. But it turns out it's correct and [just `[0]` is incorrect syntax](https://www.typescriptlang.org/play?#code/BQbwvgBAhgztB2BPAlAfgHQHsBOBLA5rvFADYBqpArgKaoDaADALoDcAUG6JLAihjgSKkKJGhkasgA).

Reply:
Fascinating. Thanks for checking!